### PR TITLE
Updated the readme 

### PR DESCRIPTION
**Pull request by lili2311**
## What are the changes included in this PR?

Updated readme with better instructions regarding storing credentials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,26 @@ Women Hack for Non-Profits website rebuild.
...
 ```
 {
-	"key": "",
-	"secret": "",
-	"bucket": "",
-	"region": ""
+	"key": "your key here",
+	"secret": "your secret here",
+	"bucket": "devwomenhackfornonprofits",
+	"region": "us-west-2"
```
Comment (tanyapowell) [inline README.md:21]: how comes the region for the `dev` bucket is different from the region for the `production` bucket?

Comment (lili2311) [inline README.md:21]: because that is what was chosen during bucket creation it wont affect anything for dev

Comment (tanyapowell) [inline README.md:21]: Fair enough

**Comment by tanyapowell** [comment]
👍 👯

**Pull request merged**